plugin/src/components/Overview/OverviewPage.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import { getKialiConfig, initKialiListeners, KialiConfig} from '../../kialiIntegration';
+import { getNamespaces, ActiveFiltersInfo, PromisesRegistry, CanaryUpgradeStatus, NamespaceInfo, OverviewCard, OverviewToolbar, nameFilter, DirectionType, OverviewDisplayMode, OutboundTrafficPolicy, ComponentStatus, OverviewType, sortFields, sortFunc, SortField } from '@kiali/core-ui';
+import { fetchCanaryStatus, fetchIstioStatus, fetchHealth, fetchTLS, fetchOutboundTrafficPolicyMode, fetchIstiodResourceThresholds, fetchValidations, fetchMetrics } from './OverviewCalls';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+const OverviewPage = () => {
+    initKialiListeners();    
```

Comment:
I think initKialiListeners() is only needed for iframes, so it is not needed anymore when calling to Kiali API.